lib/Hsm.lua

```diff
@@ -0,0 +1,121 @@
+local Fsm = require(script.Parent.Fsm)
```

Comment:
Is this the standard convention for Robase now? I seem to remember we were evaluating baste at one point

Reply:
We can use Roblox requires in lemur tests, so I don't see a downside.